src/views/Settings/changePassword.js

```diff
@@ -23,17 +21,21 @@ const ChangePassword = () => {
   const handleFormSubmit = (context, data) => {
     // TODO store and send only hashed password
     axios
-      .patch(`/api/user/${context.user.identity}`, {
-        password: data.new
-      }, {
-        headers: { Authorization: `Bearer ${context.user.accessJwt}` },
-      })
+      .patch(
+        `/api/user/${context.user.identity}`,
+        {
+          password: data.new_password,
+        },
```

Comment:
This is the payload that is being sent to the backend. 
Currently only the new_password is being sent. Please update to send all the form fields to the backend.